### ISSUE TITLE
Is anyone maintaining this project?

### ISSUE THREAD

**Issue by rjrobinson** [benmanns/goworker#41]
Is anyone maintaining this project?

**Comment by johnTurknett** [comment]
:+1:

**Comment by benmanns** [comment]
Yes, though part time and on weekends/vacations. If you (or anyone watching) would like to join the core team, feel free to get in touch.

**Comment by rjrobinson** [comment]
Actually, yes! Both John and I would Love to dive deep into this.

**Comment by RajatVaryani** [comment]
@benmanns Do you need any more people in the core team?

**Comment by benmanns** [comment]
@RajatVaryani yes! Email me at <my username>@gmail.com.

**Comment by skaurus** [comment]
I merged some popular PRs in my fork: https://github.com/skaurus/goworker
